directory/cli/src/hostgroup.py

```diff
@@ -0,0 +1,131 @@
+import click
+from collections import OrderedDict
+
+import list_command
+from list_command import field_with_same_name    	
+import ipa_wrapper_command
+import ipa_utils
+from exceptions import IpaRunError
+
+HOSTGROUP_LIST_FIELD_CONFIGS = OrderedDict([
+	('Host group name', field_with_same_name),
+	('Description', field_with_same_name),
+	('HGID', field_with_same_name),
```

Comment:
Much like with HID, HGID isn't really necessary. While IPA does assign `ipauniqueid` (`ipa hostgroup-find --all` should show you what I mean) to hostgroups these aren't as useful as the IDs which users/groups have. Can be removed.